### PR TITLE
feat: button-based embed pagination

### PR DESCRIPTION
**Pull request by suneettipirneni**
Adds button-based embed pagination functionality example shown below:
Screen Shot 2022-01-21 at 1 58 47 PM

This also updates discord.js to version 13.6.0

### CODE REVIEW

File: src/app/plugins/lionpresence.plugin.ts [hunk 1]
```diff
@@ -28,7 +29,12 @@ export default class LionPresence extends Plugin {
     }
 
     this.container.clientService.user?.setPresence({
-      activities: [{ name: activity.join(' '), type: type.toUpperCase() as ActivityType }],
+      activities: [
+        {
+          name: activity.join(' '),
+          type: type.toUpperCase() as ExcludeEnum<typeof ActivityTypes, 'CUSTOM'>,
+        },
+      ],
```
Comment (tanndlin) [inline src/app/plugins/lionpresence.plugin.ts:37]: Seems unrelated

**Review by tanndlin — COMMENTED** [review]
What happens to `MessageService.SendPagedEmbed`?
Is there anything still using it? If not just probably just remove it

**Comment by suneettipirneni** [comment]
> What happens to `MessageService.SendPagedEmbed`? Is there anything still using it? If not just probably just remove it

removed

**Comment by tanndlin** [comment]
Failing a test, and should be onto the dev branch instead of master

**Comment by tanndlin** [comment]
Was it fine when you tested it? Like no hiccups
I ran it on my PC and the second time i tried it, it crashed
But hasnt happened since

**Comment by suneettipirneni** [comment]
> Was it fine when you tested it? Like no hiccups I ran it on my PC and the second time i tried it, it crashed But hasnt happened since

Just tested it now it seems to be working fine, couldn't get it to crash

**Review by tanndlin — APPROVED** [review]
> Just tested it now it seems to be working fine, couldn't get it to crash
Yeah none of the code looks problematic so I'll just keep a mental note incase some weird stuff happens every now and then
